monitoring/README.md

```diff
@@ -193,10 +199,14 @@ applications are available at these locations by default:
 * Prometheus - Port 31090 `http://master-node.yourcluster.example.com:31090`
 * AlertManager - Port 31091 `http://master-node.yourcluster.example.com:31091`
 
-The default Grafana admin user is `admin`. Unless otherwise specified,
-the default password is randomly generated. The value is logged only
-during the initial deployment of the monitoring components and not on
-an upgrade.
+The default Grafana admin user is `admin`. Unless you used the `GRAFANA_ADMIN_PASSWORD` environment variable (either in the `user.env` file or on the command line) to specify a default password during deployment, the default password is randomly generated and displayed during deployment. 
```

Comment:
I think the word "set" is better than "used" (i.e. _Unless you set the 'GRAFANA_ADMIN_PASSWORD' environrment variable..._)

Reply:
Changed to "set"